src/bootstrap/config.go

```diff
@@ -177,8 +177,11 @@ func (config *Config) loadServerConfig() {
 		config.FactorioAdminFile = filepath.Join(config.FactorioConfigDir, config.FactorioAdminFile)
 	}
 
-	// Set random port as rconPort
-	config.FactorioRconPort = randomPort()
+	if config.FactorioRconPort == 0 {
+		config.FactorioRconPort = randomPort()
+		log.Println("Rcon port is empty, generated new one:")
+		log.Printf("Port number: %v", config.FactorioRconPort)
```

Comment:
"Port number" might not be clear what port it's referring to. I'd just combine the two log messages into one.

Reply:
You are right. Done